src/ResourceManager/Maps/Maps.Test/ScenarioTests/MapsAccountTests.ps1

```diff
@@ -0,0 +1,277 @@
+﻿# ----------------------------------------------------------------------------------
+#
+# Copyright Microsoft Corporation
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+# http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ----------------------------------------------------------------------------------
+
+<#
+.SYNOPSIS
+Test New-AzureRmMapsAccount
+#>
+function Test-NewAzureRmMapsAccount
+{
+    # Setup
+    $rgname = Get-MapsManagementTestResourceName;
+
+    try
+    {
+        # Test
+        $accountname = 'ps-' + $rgname;
+        $skuname = 'S0';
+        $location = 'West US';
```

Comment:
Changing to `$location = Get-Location -providerNamespace 'Microsoft.Maps' -resourceType 'accounts' -preferredLocation 'West US';` or `Get-Location 'Microsoft.Maps' 'accounts' 'West US';` yields this error:
```
Test Name:	Microsoft.Azure.Commands.Maps.Test.ScenarioTests.MapsAccountTests.TestGetAccounts
Test Outcome:	Failed
Result Message:	System.Management.Automation.ActionPreferenceStopException : The running command stopped because the preference variable "ErrorActionPreference" or common parameter is set to Stop: Exception of type 'System.Management.Automation.PSInvalidOperationException' was thrown.
```

Perhaps I've mis-configured something? In `Maps.Test/bin/Debug/Common.ps1`, `Get-Location` function is there, but looks like it's failing to recognize it.

Reply:
Hmm, I can't find anything in your code that would cause this error. My recommendation would be to do a git clean -xdf (after committing all local changes, as this will erase all local changes), rebuilding and rerunning the test.  I would think that something happened during all the renaming that is causing the function to not be found. Ping me if you are still having problems after this.